DEVELOPMENT.md

```diff
@@ -78,6 +78,12 @@ To run unit tests:
 ./gradlew unitTest
 ```
   
+To run connected tests:
+**DO YOU HAVE TO RUN render_config.sh for connected tests??**
```

Comment:
yes, you need to run render_config to get the credentials needed to run these tests

Reply:
Can we wrap that step into a dependent gradle task so we don't have to remember?